contrib/hack/start-server.sh

```diff
@@ -40,7 +40,7 @@ docker run -d --name apiserver \
 	-e KUBERNETES_SERVICE_PORT=6443 \
 	--privileged \
 	--net container:etcd-svc-cat \
-	scbuildimage \
+	quay.io/kubernetes-service-catalog/build-image:canary \
```

Comment:
since we allow people to change the BUILD_IMAGE_REGISTRY value during build time, for consistency, should we also pick up that value here?  But default to quay?